src/components/Nav.vue

```diff
@@ -136,3 +138,16 @@
   })
   export default class Nav extends Vue {}
 </script>
+<style scoped>
+@media screen and (max-width: 1024px) {
```

Comment:
... and so, according to my previous request, this media query is not really necessary. You can remove it. Betterwise, using max-width is not really a good idea since its behaviour can be inconsistent across browsers. Better design mobile-first and then use min-width.

Reply:
@seegwen Thanks for your feedback,  checked how the burger menu works, it seems that the appearance of the burger menu is handled by Bulma, and uses the mediaQuery max-width to make the menu appear and disappear, and the change is done at the value 1023px and not 1024 as I had corrected it the first time, I propose to keep the mediaQuery because it is consistent with the internal functioning of the menu.
If you have another solution, don't hesitate to suggest it to me.

Reply:
1023px is better to me, thanks for the fix.
I stand on my position that using a mediaquery for the menu overlay that you introduce is not necessary since it will not be visible when the burger is not clicked and the burger will not be clicked if the screen is not narrow enough.
But I will live with it. Since it does not introduce a bad bahaviour.

Thanks for your contribution.